src/backend/config/tools.py

```diff
@@ -212,6 +209,10 @@ def get_available_tools() -> dict[ToolName, dict]:
         # Retrieve name
         tool.name = tool.implementation.NAME
 
+    enabled_tools = Settings().tools.enabled_tools
```

Comment:
I think we should add docs explaining the difference between configuring the yaml and the community package

Reply:
Will add more docs around this once it is more tested - for now all the env things should still work